hack/cass-stress-gen.py

```diff
@@ -17,28 +17,36 @@
         command:
           - "/bin/bash"
           - "-c"
-          - 'cassandra-stress write no-warmup n={d.ops} cl=ONE -mode native cql3 connectionsPerHost={d.connections_per_host} -col n=FIXED\(5\) size=FIXED\(64\)  -pop seq={}..{} -node="{d.host}" -rate threads={d.threads} {d.limit} -log file=/cassandra-stress.load.data -schema "replication(factor=1)" -errors ignore; cat /cassandra-stress.load.data'
+          - 'cassandra-stress write no-warmup n={d.ops} cl=ONE -mode native cql3 connectionsPerHost={d.connections_per_host} -col n=FIXED\(5\) size=FIXED\(64\)  -pop seq={}..{} -node "{d.host}" -rate threads={d.threads} {d.limit} -log file=/cassandra-stress.load.data -schema "replication(factor=1)" -errors ignore; cat /cassandra-stress.load.data'
         resources:
           limits:
             cpu: {d.cpu}
             memory: {d.memory}
       restartPolicy: Never
       nodeSelector:
         {d.nodeselector}
+      tolerations:
+        - key: role
+          operator: Equal
+          value: cassandra-stress
+          effect: NoSchedule
       affinity:
         podAffinity:
           preferredDuringSchedulingIgnoredDuringExecution:
-            labelSelector:
-              matchLabels:
-                app: cassandra-stress
+            - weight: 100
+              podAffinityTerm:
+                topologyKey: kubernetes.io/hostname
+                labelSelector:
+                  matchLabels:
+                    app: cassandra-stress
        
 """
 
 def parse():
     parser = argparse.ArgumentParser(description='Generate cassandra-stress job templates for Kubernetes.')
     parser.add_argument('--num-jobs', type=int, default=1, help='number of Kubernetes jobs to generate - defaults to 1', dest='num_jobs')
     parser.add_argument('--name', default='cassandra-stress', help='name of the generated yaml file - defaults to cassandra-stress')
-    parser.add_argument('--scylla-version', default='2.3.1', help='version of scylla server to use for cassandra-stress - defaults to 2.3.1', dest='scylla_version')
+    parser.add_argument('--scylla-version', default='3.0.10', help='version of scylla server to use for cassandra-stress - defaults to 2.3.1', dest='scylla_version')
```

Comment:
?

Reply:
?

Reply:
Just needs a bump. I have a PR that touches this script in progress. I can do it there.

Reply:
Bumoing the default. I guess 4.0.0 or something would be better.